src/assertions/focus.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function focus(client, chai, utils) {
+    chai.Assertion.addMethod('focus', function() {
+        const selector =  utils.flag(this, 'object');
+        const negate = utils.flag(this, 'negate');
+        const immediately = utils.flag(this, 'immediately');
+
+        if (!immediately) {
+            elementExists(client, selector, negate);
```

Comment:
Actually, I think removing the entire element check would work.

Like you said, we don't care whether it exists or not...